blivet/tasks/availability.py

```diff
@@ -246,15 +285,108 @@ def available_resource(name):
+                             blockdev.CryptoTechMode.REMOVE_KEY |
+                             blockdev.CryptoTechMode.RESIZE)
+BLOCKDEV_CRYPTO = BlockDevTechInfo(plugin_name="crypto",
+                                   check_fn=blockdev.crypto_is_tech_avail,
+                                   technologies={blockdev.CryptoTech.LUKS: BLOCKDEV_CRYPTO_ALL_MODES,
+                                                 blockdev.CryptoTech.ESCROW: blockdev.CryptoTechMode.CREATE})
+BLOCKDEV_CRYPTO_TECH = BlockDevMethod(BLOCKDEV_CRYPTO)
+
+# libblockdev dm plugin required technologies and modes
+BLOCKDEV_DM_ALL_MODES = (blockdev.DMTechMode.CREATE_ACTIVATE |
+                         blockdev.DMTechMode.REMOVE_DEACTIVATE |
+                         blockdev.DMTechMode.QUERY)
+BLOCKDEV_DM = BlockDevTechInfo(plugin_name="dm",
+                               check_fn=blockdev.dm_is_tech_avail,
+                               technologies={blockdev.DMTech.MAP: BLOCKDEV_DM_ALL_MODES,
+                                             blockdev.DMTech.RAID: BLOCKDEV_DM_ALL_MODES})
+BLOCKDEV_DM_TECH = BlockDevMethod(BLOCKDEV_DM)
+
+# libblockdev loop plugin required technologies and modes
+BLOCKDEV_LOOP_ALL_MODES = (blockdev.LoopTechMode.CREATE |
+                           blockdev.LoopTechMode.CREATE |
+                           blockdev.LoopTechMode.DESTROY |
+                           blockdev.LoopTechMode.MODIFY |
+                           blockdev.LoopTechMode.QUERY)
+BLOCKDEV_LOOP = BlockDevTechInfo(plugin_name="loop",
+                                 check_fn=blockdev.loop_is_tech_avail,
+                                 technologies={blockdev.LoopTech.LOOP_TECH_LOOP: BLOCKDEV_LOOP_ALL_MODES})  # FIXME
+BLOCKDEV_LOOP_TECH = BlockDevMethod(BLOCKDEV_LOOP)
+
+# libblockdev lvm plugin required technologies and modes
+BLOCKDEV_LVM_ALL_MODES = (blockdev.LVMTechMode.CREATE |
+                          blockdev.LVMTechMode.REMOVE |
+                          blockdev.LVMTechMode.MODIFY |
+                          blockdev.LVMTechMode.QUERY)
+BLOCKDEV_LVM = BlockDevTechInfo(plugin_name="lvm",
+                                check_fn=blockdev.lvm_is_tech_avail,
+                                technologies={blockdev.LVMTech.BASIC: BLOCKDEV_LVM_ALL_MODES,
+                                              blockdev.LVMTech.BASIC_SNAP: BLOCKDEV_LVM_ALL_MODES,
+                                              blockdev.LVMTech.THIN: BLOCKDEV_LVM_ALL_MODES,
+                                              blockdev.LVMTech.CACHE: BLOCKDEV_LVM_ALL_MODES,
+                                              blockdev.LVMTech.CALCS: blockdev.LVMTechMode.QUERY,
+                                              blockdev.LVMTech.THIN_CALCS: blockdev.LVMTechMode.QUERY,
+                                              blockdev.LVMTech.CACHE_CALCS: blockdev.LVMTechMode.QUERY,
+                                              blockdev.LVMTech.GLOB_CONF: blockdev.LVMTechMode.QUERY})
```

Comment:
``GLOB_CONF`` should be checked for both ``QUERY`` and ``MODIFY``.
(I know the checks for this in *libblockdev* are not implemented strictly right now)